src/databricks/labs/ucx/hive_metastore/table_size.py

```diff
@@ -55,12 +58,19 @@ def _try_load(self) -> Iterable[TableSize]:
     def snapshot(self) -> list[TableSize]:
         """
         Takes a snapshot of tables in the specified catalog and database.
+        Return None if the table cannot be found anymore.
 
         Returns:
             list[Table]: A list of Table objects representing the snapshot of tables.
         """
         return self._snapshot(partial(self._try_load), partial(self._crawl))
 
-    def get_table_size(self, table_full_name: str) -> int:
+    def _get_safe_table_size(self, table_full_name: str) -> int | None:
```

Comment:
nit: are you getting "safe table" or are you "safely getting a table"? :)

Reply:
safely getting

Reply:
reverted to the previous name

Reply:
redash permissions are cached for up to 10 mins. It is usually faster but worst case it can be 10 mins. So after the rename operation it can take 10 mins until we can see the renamed group in the permissions. On top we have to leave some time to actually perform the rest of the test.